.circleci/config.yml

```diff
@@ -34,17 +34,6 @@ jobs:
             source /usr/local/share/virtualenvs/tap-tester/bin/activate
             stitch-validate-json tap_twitter_ads/schemas/*.json
       - add_ssh_keys
-      - run:
-          name: 'Unit Tests'
-          command: |
-            source /usr/local/share/virtualenvs/tap-twitter-ads/bin/activate
-            pip install nose coverage
-            nosetests --with-coverage --cover-erase --cover-package=tap_twitter_ads --cover-html-dir=htmlcov tests/unittests
-            coverage html
-      - store_test_results:
-          path: test_output/report.xml
-      - store_artifacts:
-          path: htmlcov
```

Comment:
@prijendev Possibly these changes are causing CCi to fail. Please revert or fix the changes.